lib/ex_aws/sqs.ex

```diff
@@ -307,6 +312,13 @@ defmodule ExAws.SQS do
     |> ExAws.Utils.camelize
   end
 
+  defp format_message_attributes(attributes) do
+    attributes
+    |> Enum.with_index
+    |> Enum.map(&format_message_attribute/1)
+    |> Enum.reduce(%{}, &Map.merge(&1, &2))
```

Comment:
Do note I use `to_string` instead of `Atom.to_string`. I think requiring atoms is unnecessary, people may want  to hook this up to dynamic sources where using atoms would be the wrong approach.